unittest/testDNGIdt.cpp

```diff
@@ -56,7 +98,9 @@ BOOST_AUTO_TEST_CASE( TestIDT_XYZToColorTemperature )
     int ret = rawProcessor.open_file( ( pathToRaw.string() ).c_str() );
     ret     = rawProcessor.unpack();
```

Comment:
do you want to put `BOOST_REQUIRE_EQUAL( ret, LIBRAW_SUCCESS );` after each line where `ret = ` is assigned? (so it would be open_file and upack. For early clear failure.